docs/user/tutorials/01-10-add-delete-modules.md

```diff
@@ -0,0 +1,28 @@
+# Adding a new Module using Kyma CLI
+This tutorial shows how you can create a new Module using Kyma CLI. 
```

Comment:
I would suggest keeping nomenclature from the title and command itself and changing the `create` to the `add` because it may be confusing for a user (this tutorial is not about making/creating a new module).